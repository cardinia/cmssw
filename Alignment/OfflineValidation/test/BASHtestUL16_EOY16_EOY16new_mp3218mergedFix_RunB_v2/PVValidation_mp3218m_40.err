27-Sep-2019 21:42:21 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/00000/B88F5648-4F07-E911-9D76-782BCB4FBF56.root
27-Sep-2019 21:42:25 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/00000/B88F5648-4F07-E911-9D76-782BCB4FBF56.root
Begin processing the 1st record. Run 273592, Event 1722080, LumiSection 1 on stream 0 at 27-Sep-2019 21:42:37.689 CEST
Begin processing the 1001st record. Run 273592, Event 27622597, LumiSection 15 on stream 0 at 27-Sep-2019 21:44:30.896 CEST
Begin processing the 2001st record. Run 273592, Event 58389052, LumiSection 30 on stream 0 at 27-Sep-2019 21:45:12.810 CEST
Begin processing the 3001st record. Run 273592, Event 88129350, LumiSection 45 on stream 0 at 27-Sep-2019 21:45:50.083 CEST
Begin processing the 4001st record. Run 273592, Event 117804587, LumiSection 60 on stream 0 at 27-Sep-2019 21:46:28.691 CEST
Begin processing the 5001st record. Run 273592, Event 145302031, LumiSection 74 on stream 0 at 27-Sep-2019 21:47:10.279 CEST
Begin processing the 6001st record. Run 273592, Event 176118318, LumiSection 89 on stream 0 at 27-Sep-2019 21:47:50.002 CEST
Begin processing the 7001st record. Run 273592, Event 206363035, LumiSection 104 on stream 0 at 27-Sep-2019 21:48:31.860 CEST
Begin processing the 8001st record. Run 273592, Event 235670972, LumiSection 119 on stream 0 at 27-Sep-2019 21:49:11.314 CEST
Begin processing the 9001st record. Run 273592, Event 265508728, LumiSection 134 on stream 0 at 27-Sep-2019 21:49:50.903 CEST
Begin processing the 10001st record. Run 273592, Event 293042828, LumiSection 148 on stream 0 at 27-Sep-2019 21:50:28.652 CEST
Begin processing the 11001st record. Run 273592, Event 322987969, LumiSection 163 on stream 0 at 27-Sep-2019 21:51:05.777 CEST
27-Sep-2019 21:51:42 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/00000/B88F5648-4F07-E911-9D76-782BCB4FBF56.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[7.673MB/7.673MB][100%][==================================================][7.673MB/s]  [7.673MB/7.673MB][100%][==================================================][7.673MB/s]  
