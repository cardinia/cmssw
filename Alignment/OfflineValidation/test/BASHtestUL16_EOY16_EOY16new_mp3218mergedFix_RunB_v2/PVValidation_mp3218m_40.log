000 (3196549.040.000) 09/27 21:31:59 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196549.040.000) 09/27 21:41:47 Started transferring input files
	Transferring to host: <188.184.142.51:9618?addrs=188.184.142.51-9618+[2001-1458-301-46--100-42]-9618&noUDP&sock=4581_7a93_13378>
...
040 (3196549.040.000) 09/27 21:41:47 Finished transferring input files
...
001 (3196549.040.000) 09/27 21:41:47 Job executing on host: <188.184.142.51:9618?addrs=188.184.142.51-9618+[2001-1458-301-46--100-42]-9618&noUDP&sock=2397_b89a_3>
...
006 (3196549.040.000) 09/27 21:41:56 Image size of job updated: 34636
	20  -  MemoryUsage of job (MB)
	19940  -  ResidentSetSize of job (KB)
...
006 (3196549.040.000) 09/27 21:46:56 Image size of job updated: 543456
	528  -  MemoryUsage of job (MB)
	540508  -  ResidentSetSize of job (KB)
...
040 (3196549.040.000) 09/27 21:52:10 Started transferring output files
...
040 (3196549.040.000) 09/27 21:52:10 Finished transferring output files
...
005 (3196549.040.000) 09/27 21:52:10 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:08:22, Sys 0 00:00:06  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:08:22, Sys 0 00:00:06  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	4180  -  Run Bytes Sent By Job
	1021  -  Run Bytes Received By Job
	4180  -  Total Bytes Sent By Job
	1021  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     7900        1   1231220 
	   Memory (MB)          :      528     2000      2000 
...
