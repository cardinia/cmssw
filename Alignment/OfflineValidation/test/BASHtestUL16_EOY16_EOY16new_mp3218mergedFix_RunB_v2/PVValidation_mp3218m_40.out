Job started at  Fri Sep 27 21:41:48 CEST 2019
batch dir: /pool/condor/dir_26843 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_273592_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 7.8M -rw-r--r--. 1 tkello zh 7.7M Sep 27 21:52 PVValidation_mp3218m_273592.root -rw-r--r--. 1 tkello zh 20K Sep 27 21:41 PVValidation_mp3218m_273592_cfg.py -rw-r--r--. 1 tkello zh 2.9K Sep 27 21:52 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 27 21:42 _condor_stdout -rwxr-xr-x. 1 tkello zh 1021 Sep 27 21:41 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:42 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:41 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:41 var
Job ended at  Fri Sep 27 21:52:10 CEST 2019
